nbformat/v4/nbformat.v4.schema.json

```diff
@@ -59,6 +59,14 @@
                     "description": "Original notebook format (major number) before converting the notebook between versions. This should never be written to a file.",
                     "type": "integer",
                     "minimum": 1
+                },
+                "title": {
+                    "description": "The title of the notebook document",
+                    "type": "string"
+                },
+                "author": {
+                    "description": "The author(s) of the notebook document",
+                    "type": "string"
```

Comment:
Especially if we are tackling Latex Export, I think it should be a list of object. Authors can have emails, phones affiliations, and be corresponding authors. Making it objects, leave it flexible.